tests/test_augmentation.py

```diff
@@ -0,0 +1,35 @@
+#!/usr/bin/env python
+# -*- coding: utf-8 -*-
+
+"""
+test_augmentation
+----------------------------------
+
+Tests the ``augmentation`` module.
+"""
+
+import pytest
+
+from mindmeld.augmentation import EnglishParaphraser
+
+
+@pytest.fixture(scope="module")
+def english_paraphraser(kwik_e_mart_app_path):
+    return EnglishParaphraser(kwik_e_mart_app_path)
+
+
+@pytest.mark.extras
+@pytest.mark.augment
+@pytest.mark.parametrize(
+    "query, num_paraphrases",
+    [
+        ("some text", 10),
+        ("another text", 5),
+        ("yet another text", 2),
+    ],
+)
+def test_num_paraphrases(english_paraphraser, query, num_paraphrases):
```

Comment:
Can we also test for the results to make sure they are deterministic and make sense?